<?xml version="1.0" encoding="utf-8"?>
<testsuite errors="0" failures="1" name="TestAndroidBasic" tests="2" time="0.000">
	<testcase classname="TestAndroidBasic" name="testA_Success" time="0.000"/>
	<testcase classname="TestAndroidBasic" name="testB_Failure" time="0.000">
		<failure message="" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/Users/obartley/Projects/SeeMonkey/example/scripts/sanity.sikuli/sanity.py", line 27, in testB_Failure
    assert False
AssertionError
]]>		</failure>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
